include/nodejs/node_osrm_support.hpp

```diff
@@ -845,6 +848,9 @@ inline bool parseCommonParameters(const v8::Local<v8::Object> &obj, ParamType &p
                     Nan::ThrowError("this 'annotations' param is not supported");
                     return false;
                 }
+
+                params->annotations =
```

Comment:
I would remove this `annotations` boolean field completely, but it seems it would break libosrm backward compatibility

Reply:
We should think about all the breaking changes we would make for a v6 release.

Reply:
Heh, it's already there: https://github.com/Project-OSRM/osrm-backend/issues/3644